docs/user-guide/cli-configure-daemon-on-zlinux-os.md

```diff
@@ -34,18 +37,22 @@ The following steps describe how to install and build the daemon binary on z/Lin
 
    After the command completes successfully, the Zowe daemon binary is a file named `zowe` that can be found in the `target/release` directory.
 
-7. Copy the binary to another location on the system and add it to your PATH.
-8. (Optional) Modify the file permissions to allow others to use the same binary:
+7. Copy the binary to another location on the system and add that location **[is "that location" correct?]** to your PATH.
+8. To allow others to use the same binary, modify the file permissions: **[where is this modification made? The binary?]**
```

Comment:
@adam-wolfe @zFernand0 - please respond

Reply:
Yes. They need to modify the permissions of the binary to allow other users or groups to execute it.

Reply:
`-8` The file permission changes are performed "against" (`¯\_(ツ)_/¯`) the file. 

For example: 
```
1:    zFernand0 @ SSH:~/test# touch test.file

2:    zFernand0 @ SSH:~/test# ls -al 
3:    total 8
4:    drwxr-xr-x 2 root root 4096 Feb 14 22:12 .
5:    drwxr-xr-x 3 root root 4096 Feb 14 22:12 ..
6:    -rw-r--r-- 1 root root    0 Feb 14 22:12 test.file

7:    zFernand0 @ SSH:~/test# chmod +x test.file 

8:    zFernand0 @ SSH:~/test# ls -al 
9:    total 8
10:   drwxr-xr-x 2 root root 4096 Feb 14 22:12 .
11:   drwxr-xr-x 3 root root 4096 Feb 14 22:12 ..
12:   -rwxr-xr-x 1 root root    0 Feb 14 22:12 test.file
```

The default permissions of the file (after creation) are `-rw-r--r--` (everyone can read it, but only the owner can write to it

And line 7 changes the file permissions so that anyone can execute it (`-rwxr-xr-x`) 😋